pkg/operation/botanist/component/dependencywatchdog/dependency_watchdog_test.go

```diff
@@ -399,10 +415,18 @@ status: {}
 				}))
 
 				Expect(c.Get(ctx, client.ObjectKeyFromObject(managedResourceSecret), managedResourceSecret)).To(Succeed())
+				fmt.Println("managed resource output")
+				fmt.Println(managedResource)
```

Comment:
Remove those `fmt.Println` statements

Reply:
Sorry for these, I've removed them already with a force commit. I wanted to avoid this being spotted but realised you were fast here. I understand the protocol to add review comments adaptations as separate commits.  I'll keep in mind for next occurrence.